pom.xml

```diff
@@ -20,38 +20,77 @@
 
   <groupId>com.datastax.oss</groupId>
   <artifactId>java-driver-shaded-guava</artifactId>
-  <version>25.2-jre-SNAPSHOT</version>
-  <packaging>bundle</packaging>
+  <version>25.1-jre-graal-sub-1-SNAPSHOT</version>
```

Comment:
The version looks OK to me, hopefully there won't be other substitutions any time soon.

I was briefly wondering if we should also use the opportunity to upgrade the Guava version (latest is 29.0), but I don't think so. We only use this internally, 25.1 works for the driver (no known bugs or missing features), so let's keep this PR simple.

Reply:
:+1: to your analysis @olim7t .  Seems like a Guava upgrade would probably need a ticket of it's own as those usually involve a non-trivial amount of, er, "fun"